wcivf/apps/people/models.py

```diff
@@ -477,6 +477,18 @@ def show_national_manifesto(self):
         except AttributeError:
             return False
 
+    @property
+    def has_additional_manifesto(self) -> PersonPost:
+        if self.featured_candidacy.post_election.ballot_paper_id in [
+            "parl.aberdeenshire-north-and-moray-east.2024-07-04"
+        ] and self.featured_candidacy.party_name in [
+            "Labour Party",
```

Comment:
I think party IDs will be better here, and we'll want to consider Labour and Co-Op too